cleanup_utils/integration_cleanup.py

```diff
@@ -31,13 +31,17 @@
 from pytz import UTC
 
 from cleanup_utils.aws_cleanup import AwsCleanupInstance
-from cleanup_utils.openstack_cleanup import OpenStackCleanupInstance
 from cleanup_utils.dns_cleanup import DnsCleanupInstance
+from cleanup_utils.mysql_cleanup import MySqlCleanupInstance
+from cleanup_utils.openstack_cleanup import OpenStackCleanupInstance
 
 
 # Constants ###################################################################
 
-DEFAULT_AGE_LIMIT = datetime.utcnow().replace(tzinfo=UTC) - timedelta(days=3)
+# Default age at which things should be cleaned up, in days
+DEFAULT_AGE_LIMIT = 3
+DEFAULT_CUTOFF_TIME = (
+    datetime.utcnow().replace(tzinfo=UTC) - timedelta(days=DEFAULT_AGE_LIMIT))
```

Comment:
@jamestait, don't you think the below style of brackets is more readable and looks cleaner?

```python
DEFAULT_CUTOFF_TIME = (
    datetime.utcnow().replace(tzinfo=UTC) - timedelta(days=DEFAULT_AGE_LIMIT)
)
```

Reply:
Yes, thank you - fixed this and the other instances. :)